packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/index.scss

```diff
@@ -0,0 +1,365 @@
+.load-strategy {
+    &__wrapper {
+        position: fixed;
+        top: 40px;
+        z-index: 10;
+        width: 100%;
+
+        .dc-mobile-full-page-modal {
+            @include mobile {
+                width: 100%;
+            }
+
+            &__body {
+                height: 100%;
+            }
+        }
+
+        & .dc-tabs {
+            @include mobile {
+                height: 100%;
+                display: unset;
+                flex: 1;
+
+                &__content {
+                    height: calc(100% - 40px);
+                }
+            }
+        }
+    }
+
+    &__container {
+        @include desktop {
+            height: calc(80vh - 0.8rem - 4rem - 4.8rem - 5.8rem);
+
+            &--has-footer {
+                height: calc(80vh - 0.8rem - 4rem - 4.8rem - 5.8rem - 7.4rem);
+            }
+
+            > * {
+                height: 100%;
+                margin-top: 0.8rem;
+            }
+        }
+
+        @include tablet {
+            height: calc(80vh - 0.8rem - 4rem - 4.8rem - 5.8rem);
+
+            &--has-footer {
+                height: calc(80vh - 0.8rem - 4rem - 4.8rem - 5.8rem - 7.4rem);
+            }
+
+            > * {
+                height: 100%;
+                margin-top: 0.8rem;
+            }
+        }
+
+        @include mobile {
+            height: 100%;
+        }
+    }
+
+    &__title {
+        font-size: var(--text-size-s);
+        font-weight: 700;
+        margin: 1.5rem;
+    }
+
+    &__preview-workspace {
+        padding: 1.5rem;
+        border-radius: $BORDER_RADIUS;
+        border: solid 1px var(--border-normal);
+        height: calc(100% - 30rem); // TODO: What is 37px?
+        position: relative;
+        overflow: hidden;
+        margin-bottom: 1.6rem;
+        margin-right: 1.6rem;
+
+        &-controls {
+            padding: 0.7rem 0.5rem;
+            display: flex;
+            flex-direction: column;
+            position: absolute;
+            right: 1.6rem;
+            bottom: 1.6rem;
+            border-radius: 3rem;
+            background-color: $color-grey-2;
```

Comment:
no need to add var?

Reply:
we can add var but i think we can also use it without var in scss.